packages/web-components/src/image/image.stories.ts

```diff
@@ -0,0 +1,249 @@
+import { html } from '@microsoft/fast-element';
+import type { Args, Meta } from '@storybook/html';
+import { renderComponent } from '../helpers.stories.js';
+import type { Image as FluentImage } from './image.js';
+import { ImageBorderRadius, ImageFit, ImageShape } from './image.options.js';
+import './define.js';
+
+type ImageStoryArgs = Args & FluentImage;
+type ImageStoryMeta = Meta<ImageStoryArgs>;
+
+const imageTemplate = html<ImageStoryArgs>`
```

Comment:
I am not sure I understand the default story:
https://fluentuipr.z22.web.core.windows.net/pull/26936/web-components/storybook/index.html?path=/docs/components-image--image

Reply:
I was simply showing how different images would respond to attribute selection. I will reduce the first example to just one img.